MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -174,9 +174,6 @@ public class CarPlayNavigationViewController: UIViewController {
     public override func viewSafeAreaInsetsDidChange() {
         super.viewSafeAreaInsetsDidChange()
         
-        let navigationBarIsOpen = view.safeAreaInsets.top > 0
-        mapView?.compassView.isHidden = navigationBarIsOpen
```

Comment:
This is hidden by default?

Reply:
... yes, https://github.com/mapbox/mapbox-navigation-ios/blob/9556c055d01d0fc7f8ad9205e96bf47d91d0f248/MapboxNavigation/CarPlayNavigationViewController.swift#L113